### PR TITLE
The Opaopapanerd Statue

### PR DESCRIPTION
**Pull request by g-ramalho**
## About

Adds the opao statue in memory of our fallen heroes

## Changelog
:cl:
add: Apud memoriae de... Adds the opao statue in memory of our fallen heroes.
/:cl:

### CODE REVIEW

**Comment by g-ramalho** [comment]
it's over

**Comment by g-ramalho** [comment]
or is it?...

File: beatstation/obj/structures/statues.dm [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/obj/structure/statue/sandstone/asstue
+	name = "opao statue"
+	desc = "THE GREAT PURGE - March 25, 2020."
+	icon = 'hippiestation/icons/obj/statueopao.dmi'
```
Comment (Neotwr) [inline beatstation/obj/structures/statues.dm]: `beatstation/icons/obj/statueopao.dmi`

**Review by Neotwr — COMMENTED** [review]
Also, fix your changelog with something more descriptive

**Review by ghost — CHANGES_REQUESTED** [review]
Replace all "obj/structure/statue/sandstone/asstue" by "obj/structure/statue/sandstone/opao" and that's it (I guess.)

**Comment by g-ramalho** [comment]
finally some sleep

<@209038200008736768>

**Pull request merged**